### DISCUSSION TITLE
Loupedeck Support

### DISCUSSION THREAD

**Discussion by Nexow91** [Palakis/obs-websocket#1250] [Ideas]
Hello everyone, 

First of all, thank you very much for your great work. 
I use OBS NDI via a VM unfortunately my Loupdeck CT can only control OBS natively on the main computer. Would it be possible that you write a plugin for Loupdeck then I could control the remote OBS on the streaming VM via websocket that would be mega. 

https://loupedeck.com/de/products/loupedeck-ct/
https://loupedeck.com/loupedeck-plugins/

Regards

**Comment by tt2468** [comment]
We don't maintain integrations for third-party products. It is generally accepted that third-party products will implement the obs-websocket protocol if they want to have a proper integration with OBS. I do see a changelog on their website which references adding support for the v5 protocol when we released OBS 28, so I don't see why it wouldn't be able to connect to obs-websocket on a different computer: https://support.loupedeck.com/update-release-5.2.2-obs-is-now-available

**Comment by Nexow91** [comment]
Because the connection is apparently only established using ‘local’. If I open OBS and Loupdeck on the source computer, yes, I have a connection but not to another PC. 
I can't find any settings for other IP addresses or ports.

**Reply by tt2468** [reply]
I would highly suggest opening some kind of support ticket with them, and asking if they can add a remote IP field if there isn't one hidden somewhere. It might just be a simple UI oversight on their part.
